client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -80,6 +82,14 @@ const SitePlanIcon = styled.div`
 export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 	const { __ } = useI18n();
 	const translatedStatus = useSiteLaunchStatusLabel( site );
+	const { ref, inView } = useInView();
+	const [ inViewOnce, setInViewOnce ] = useState( false );
+
+	useEffect( () => {
+		if ( inView ) {
+			setInViewOnce( true );
+		}
+	}, [ inView, setInViewOnce ] );
```

Comment:
I've discovered we can pass [the option](https://www.npmjs.com/package/react-intersection-observer#user-content-options) `triggerOnce` to `useInView`. So we can remove our implementation of `inViewOnce`.

```suggestion
	const { ref, inView: inViewOnce } = useInView( { triggerOnce: true } );
```

Reply:
Thanks for the suggestion, @sejas ! Implemented with e741fe44a0